modules/search/src/main/java/com/sygic/maps/module/search/SearchFragment.kt

```diff
@@ -166,9 +167,11 @@ class SearchFragment : Fragment(), SdkInitializationManager.Callback, SearchResu
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        val resultListAdapter = SearchResultListAdapter<SearchResult>()
```

Comment:
Tu je stále jen ta "drobnost", že se každým `onCreate` vytvoří nový, ale použije se jen poprvé :(

Reply:
ano, neprisiel som na to, ako by sa to dalo spravit bez toho, kedze `factory.with()` sa zavola vzdy